### DISCUSSION TITLE
What is XOpenGL?

### DISCUSSION THREAD

**Discussion by jharrison03** [OldUnreal/UnrealTournamentPatches#1078] [General]
Sorry if this sounds like a stupid question.  What is XOpenGL and should I be using this instead of the default OpenGL renderer?.  AFAIK, there's no explanation of exactly what this is.

**Comment by jharrison03** [comment]
Obviously the title should read "XOpenGL", not "XOpelGL".

**Comment by SeriousBuggie** [comment]
It is just another render which used OpenGL technology. It is just different.
Use what you more like, or what you found better for you, or what better suits for your needs.
